src/Expr.cc

```diff
@@ -42,6 +42,8 @@ const char* expr_name(ExprTag t) {
         "-",
         "+",
         "-",
+        "add ",
```

Comment:
Maybe add a comment to the header of this method that explains why some of them have spaces.

Reply:
Make sense, will do

Reply:
Now pushed.